actions.yaml

```diff
@@ -19,3 +19,54 @@ generate-self-signed-certificate:
   required:
     - common-name
   additionalProperties: False
+
+get-outstanding-certificate-requests:
+  description: >-
+    Get list of all outstanding certificate signing requests.
+
+get-certificate-request:
+  description: >-
+    Get certificate requests from a specific requirer application.
+  params:
+    relation-id:
+      type: string
+      description: >-
+        ID of the relation between the tls-certificate-operator and the requirer.
+  required:
+    - relation-id
+
+provide-certificate:
+  description: >-
+    Provide a certificate to a specific requirer unit.
+  params:
+    relation-id:
+      type: string
+      description: >-
+        ID of the relation between the tls-certificate-operator and the requirer.
+    certificate-signing-request:
+      type: string
+      description: >-
+        The request to which the certificate is being provided.
+    certificate:
+      type: string
+      description: >-
+        Base64 encoded TLS certificate (do not use if 'generate-self-signed-certificates' is set to true).
+    ca-chain:
+      type: string
+      description: >-
+        Base64 encoded CA chain (do not use if 'generate-self-signed-certificates' is set to true).
+        If not provided, the ca-chain will be composed of the application and the ca-certificate.
+    ca-certificate:
+      type: string
+      description: >-
+        Base64 encoded CA Certificate (do not use if 'generate-self-signed-certificates' is set to true).
+    unit-name:
+      type: string
+      description: >-
+        Name of the unit to which the certificate is being provided.
+  required:
+    - relation-id
+    - certificate-signing-request
+    - certificate
+    - ca-chain
```

Comment:
Is "ca-chain" really required? In parameter description (line 58) it seems that we can concatenate "certificate" and "ca-certificate" parameters if not set.

Reply:
I removed that line, for now I'd say it should be mandatory.